### ISSUE TITLE
Multiple entries for same molecule type in [ system ] break G->L conversion

### ISSUE THREAD

**Issue by ctk3b** [shirtsgroup/InterMol#310]
`[ system ]` directives that contain multiple entries for the same molecule type, e.g.,

```
SOL 10
SOL  5
```

end up writing incorrect bond/angle/dihedral indices in the LAMMPS output.

**Comment by mrshirts** [comment]
Any immediate thoughts for what the basic issue is?  There are two ways to do it: treat them as literally different molecules, or try to merge them.  Errors like this would occur if what is being done is halfway in between.

**Comment by ctk3b** [comment]
Yeah it's an accounting error somewhere involving the `offset` variable. Will try to pin it down later today - was mostly documenting this as it was a user reported error via email.

**Comment by mrshirts** [comment]
Great, sounds good.

**Comment by dspoel** [comment]
Merging might be tricky when the order it alternating, e.g.
SOL 10
Protein 1
SOL 5
Na+ 3
Cl - 5

**Comment by mrshirts** [comment]
Is this holding you guys up right now or is there a workaround?  We are working on prioritizing the existing issues!
